src/shared/services/bolt/bolt.js

```diff
@@ -35,8 +36,7 @@ import BoltWorkerModule from 'worker-loader?inline!./boltWorker.js'
 /* eslint-enable import/no-webpack-loader-syntax */
 
 let connectionProperties = null
-let boltWorkerRegister = {}
-let cancellationRegister = {}
+let boltWorkPool = new WorkPool(() => new BoltWorkerModule(), 10)
```

Comment:
FYI if we choose to surface the worker pool as `:config` we will probably need to only allow worker pools `> 0` as a pool of `0` is always in a `BUSY` state

Reply:
Yeah, 0 would be infinite imo. 
New ones won't be recreated as long as there are existing free ones anyway.